Sources/SwiftSDKGenerator/Artifacts/DownloadableArtifacts.swift

```diff
@@ -121,3 +133,10 @@ struct DownloadableArtifacts: Sendable {
     )
   }
 }
+
+struct StringError: Error, CustomStringConvertible {
```

Comment:
I don't see any value in stringly typed APIs in Swift, especially for errors. These are harder to test against, and go against the grain of the static typing approach that Swift takes. IMO a nested `enum Error` within `DownloadableArtifacts` with explicit cases for each error path would work, and would make it easier to adopt typed throws in the future.

Reply:
Done, added an error enum.